### PR TITLE
Hai Reveal conversations: reduce text walls in missions A12 & A13

### PR DESCRIPTION
**Pull request by UnorderedSigh**
**Feature:** This reorders dialog and adds choices to split up text walls in missions A12 and A13 in Hai Reveal.

Note: My work to reduce text walls has been split into smaller PRs by @MasterOfGrey's request, to make them easier to review.

## Feature Details
Conversations with especially long text are likely to be ignored, or at best skimmed, by players. That will make the Hai Reveal more confusing. This PR simplifies the text by:

1. In A12, the player can interrupt Turner sooner, avoiding a rant about a movie. Details of the movie aren't important; they're more of a developer inside joke at this point. I strategically moved the interruption to just after he mentions Quarg and Korath, so the player may be interested enough to read the rest of his rant.
2. The player can skip the election discussion in A13 and go straight to the results.
3. A few well-placed choices to break up other text walls.

## Testing Done
Replayed A12 and A13 a lot.

[Friction Diction~3026-01-06 TRY7 before A12.txt](https://github.com/endless-sky/endless-sky/files/9998069/Friction.Diction.3026-01-06.TRY7.before.A12.txt)

### CODE REVIEW

**Comment by UnorderedSigh** [comment]
Pinging @bene-dictator hoping for a review on this. The changes to dialog are minimal; mostly I just rearranged things.

File: data/hai/hai reveal 4 middle.txt
Comment (bene-dictator) [inline data/hai/hai reveal 4 middle.txt:1340]:
```suggestion
			`	After the others are gone, Sayari explains the situation. "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change, who was also just elected. They are remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now. You're to come with us too, and I'm told I'm required to travel with you, as insurance."`
```

File: data/hai/hai reveal 4 middle.txt
Comment (bene-dictator) [inline data/hai/hai reveal 4 middle.txt:1288]:
```suggestion
			`You are granted access to the secure spaceport, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day, and there is a gloom developing as Ya Hai disappears behind the buildings of the capital, when you enter the diplomatic wing of the government complex. Fortunately, though the place appears nearly deserted, you swiftly run into Alondo and Remington who are sitting in the empty cafeteria with paired glasses of what appears to be scotch whiskey.`
```

Comment (bene-dictator) [reply data/hai/hai reveal 4 middle.txt:1288]: Actually not too sure about this one.

**Comment by UnorderedSigh** [comment]
I modified the dialog to remove all "action choices" at Amazinite's request.

File: data/hai/hai reveal 4 middle.txt
Comment (bene-dictator) [inline data/hai/hai reveal 4 middle.txt:1267]:
```suggestion
				`	"I'll go to my ship's comms now and call him."`
```

File: data/hai/hai reveal 4 middle.txt
Comment (roadrunner56) [inline data/hai/hai reveal 4 middle.txt:1345]: Having another choice here might be good.
```suggestion
				`	"What is it, Sayari?"`
				`	(Follow her.)`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/hai/hai reveal 4 middle.txt
Comment (UnorderedSigh) [reply data/hai/hai reveal 4 middle.txt:1345]: You would already have followed her by the time you ask, "What is it, Sayari?"

I do need a second option, though. I will ponder it. Either that, or I'll put the paragraph back together.

Comment (UnorderedSigh) [reply data/hai/hai reveal 4 middle.txt]: Okay. There are two options now. How do you like them?

**Review by roadrunner56 — APPROVED** [review]

**Review by EjoThims — APPROVED** [review]

File: data/hai/hai reveal 4 middle.txt
Comment (MasterOfGrey) [inline data/hai/hai reveal 4 middle.txt:1358]:
```suggestion
			`	When you finish on the possibility of an Unfettered war fleet appearing at the wormhole to scour pirate worlds, he stops you with a pained expression. "Well, it's all falling apart, isn't it... Alright, look, this is above my pay-grade, and I don't think within Terry's either. You need to go contact Admiral Danforth. The connection from here to Farpoint is excellent now, since all our other traffic is routing through those new relays, so it should be easy enough. I'll make sure no one does anything stupid from this side. I'm sure there's a solution here, we all just need to keep our heads and call the right people."`
```
My bad

File: data/hai/hai reveal 4 middle.txt
Comment (MasterOfGrey) [inline data/hai/hai reveal 4 middle.txt:1382]: I do think this bit of clarity is relevant, since you'd expect to be blocked right now. Otherwise the elegance of these edits got me thinking on a slightly different path about how to word this though so I hope this sounds good. :)
```suggestion
			`You are granted access to the secure spaceport, potentially because nobody has filed any paperwork to stop you, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day and the developing gloom as Ya Hai sinks behind the buildings of the capital underscores the mood when you enter the diplomatic wing of the government complex. The place appears nearly deserted.`
			`	Fortunately though, you swiftly discover Alondo and Remington who are sitting in the empty cafeteria with paired glasses of what appears to be scotch whiskey. When he sees you Alondo raises a hand and gestures for you to sit with them. "<first>! Bloody hell, come join us."`
```

File: data/hai/hai reveal 4 middle.txt
Comment (MasterOfGrey) [inline data/hai/hai reveal 4 middle.txt:1396]:
```suggestion
				`	"Can either of you explain all of this?"`
```

File: data/hai/hai reveal 4 middle.txt
Comment (MasterOfGrey) [inline data/hai/hai reveal 4 middle.txt:1410]: They are intentionally alone, abandoned by the Hai who are dealing with their own concerns. They've retrieved a bottle from somewhere and you'd have to get yourself a glass, though you can probably work out how to pour yourself water from behind the bar easily enough too.

File: data/hai/hai reveal 4 middle.txt
Comment (MasterOfGrey) [inline data/hai/hai reveal 4 middle.txt:1444]:
```suggestion
			`	She explains, "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change, who was also just elected. They are remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now. You're to come with us too, and I'm told I'm required to travel with you, as insurance."`
```

**Review by MasterOfGrey — CHANGES_REQUESTED** [review]
Only a handful of spots I think need something.

File: data/hai/hai reveal 4 middle.txt
Comment (UnorderedSigh) [reply data/hai/hai reveal 4 middle.txt]: I changed it so Alondo offers to get you a drink.

File: data/hai/hai reveal 4 middle.txt
Comment (MasterOfGrey) [inline data/hai/hai reveal 4 middle.txt:1591]:
```suggestion
			`	Alondo wobbles over to the bar and says, "I'll get you a drink, <last>. What'll you have?"`
```
he ain't sober

**Review by MasterOfGrey — APPROVED** [review]

**Comment by bene-dictator** [comment]
I'll look through this one last time.

**Comment by Saugia** [comment]
I mentioned this on discord, but @bene-dictator @MasterOfGrey, you have to find a middle ground with these reviews. For example, if Grey hits "Approved", that's it - no more changes, no more reviews. As it currently stands, if he hits approve, and you decide to review after (which this has happened many times throughout many PRs), procedure dictates he has to review again. At this rate, you guys are just circling each other, and it's getting to be a bit frustrating to be told or base things on these PRs being ready, but then they aren't, for the 5th time in a row.

**Comment by bene-dictator** [comment]
I'm only looking for spelling errors.

File: data/hai/hai reveal 4 middle.txt
Comment (bene-dictator) [inline data/hai/hai reveal 4 middle.txt:1562]:
```suggestion
			`You are granted access to the secure spaceport, potentially because nobody has filed any paperwork to stop you, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day when you enter the diplomatic wing of the government complex. The developing gloom as Ya Hai sinks behind the buildings of the capital underscores the mood; the place appears nearly deserted.`
```
This sentence is way too long again.

File: data/hai/hai reveal 4 middle.txt
Comment (bene-dictator) [inline data/hai/hai reveal 4 middle.txt:1589]:
```suggestion
			`	Remington answers your question. "That Teeneep of yours, she had her position settled yesterday. She's being..." He curls his lip and frowns while looking for the right word. "Inaugurated! She's being inaugurated in about an hour or so. It's why we're still here. She's the youngest 'elder' since before we developed fire, or some similarly ridiculous time-frame. She seems to have the most influence among the new lot. She's a firebrand, that one, but she seems principled and decent. We're waiting to hear what she has to say in her victory speech.`
```
That is all, I think.

**Review by bene-dictator — APPROVED** [review]

**Comment by UnorderedSigh** [comment]
@Saugia I'm the one who has to apply the changes, so any post-MasterOfGrey changes are on me. I limit them to minor textual corrections. If anyone notices major problems, I'll ask @MasterOfGrey for a re-review. He hasn't complained to me about it, so I have assumed he is okay with this approach.

**Review by MasterOfGrey — APPROVED** [review]
~~I'm ok with it unless people ask me to come back here again afterwards.~~

If it is indeed *only* grammar edits, or rearranging the syntax of a sentence, please just make a comment saying so after merging and people can consider my approving review to still be valid.
If the actual words have changed then I gotta come back...

**Comment by UnorderedSigh** [comment]
The worst of all is the commas. Everyone always disagrees about where the commas belong.

**Comment by Saugia** [comment]
As long as they are *just* grammar edits, yes. No "whoops rewriting this"; that's when you *don't* click the approve button. We had to tell the same thing to our reviewer team recently. Once you click that approval button, that's it. Please no more additional changes after that.

**Pull request merged**
